db/schema.sql

```diff
@@ -0,0 +1,609 @@
+-- schema definition
+
+#########################
+#
+# DDL Section
+#
+#########################
+
+#  OfferingType
+
+DROP TABLE IF EXISTS OfferingType;
+
+CREATE TABLE IF NOT EXISTS OfferingType (
+  id          INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
+  name        VARCHAR(255) COMMENT 'starbucks refreshers beverages',
+  description VARCHAR(255) COMMENT 'longer days mean more time to sip something warm'
```

Comment:
this is minor but can we increase the varchar(255) to something bigger? some of the descriptions in the starbucks can be longer than 255.

Reply:
Ombre Pink Drink for example has a description thats pretty long